lib/chef/context.rb

```diff
@@ -0,0 +1,104 @@
+# freeze_string_literal: true
+#
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "licensing_config"
+
+class Chef
+  class Context
+    class << self
+      # The test kitchen will generate a nonce and sign it with a secret key.
```

Comment:
we should note here that this is only true with Progress' forked "Enterprise Test Kitchen", and the test kitchen the rest of the world uses...